### DISCUSSION TITLE
Show some mod settings in tooltip even when unmodified

### DISCUSSION THREAD

**Discussion by cl8n** [ppy/osu#29670] [Ideas]
example with AC
![image](https://github.com/user-attachments/assets/45e13b92-a12f-41a9-ac11-97a80c863979)

it gives you no info about what the minimum accuracy was, and I feel like this mod will never be so ubiquitous that everyone just knows what the default setting is.

this same logic can probably apply to more mods' settings, I have not thoroughly checked yet tho

**Comment by peppy** [comment]
Probably need an attribute/flag which allows hinting to show defaults for some settings.
